src/JasperFx.RuntimeCompiler/AssemblyGenerator.cs

```diff
@@ -215,15 +214,16 @@ internal interface ILamarAssemblyLoadContext
 		Assembly LoadFromAssemblyPath(string assemblyName);
 	}
 
-#if !NET461
+
 	public sealed class CustomAssemblyLoadContext : AssemblyLoadContext, ILamarAssemblyLoadContext, IDisposable
 	{
+		//Should this be enabled in .NET 6+?
```

Comment:
The .NET 5 conditional below would've been previously enabled with the old targets, is enabling unload something you want on .NET 6+?